dask/dataframe/core.py

```diff
@@ -5054,8 +5054,17 @@ def eval(self, expr, inplace=None, **kwargs):
         meta = self._meta.eval(expr, inplace=inplace, **kwargs)
         return self.map_partitions(M.eval, expr, meta=meta, inplace=inplace, **kwargs)
 
+    if PANDAS_GT_150:
```

Comment:
This clause is kind of odd, in that it attaches `how_default` and `thresh_default` to `_Frame` as class variables, which I don't think is what you intended. Probably cleaner to keep this logic at the module scope.

Reply:
whoops, yes you are right, I'll leave outside the class.